src/js/components/RangeInput/RangeInput.js

```diff
@@ -1,26 +1,49 @@
-import React, { forwardRef, useState } from 'react';
+import React, { forwardRef, useContext, useEffect, useState } from 'react';
 
+import { FormContext } from '../Form/FormContext';
 import { StyledRangeInput } from './StyledRangeInput';
 
-const RangeInput = forwardRef(({ onFocus, onBlur, ...rest }, ref) => {
-  const [focus, setFocus] = useState();
-  return (
-    <StyledRangeInput
-      ref={ref}
-      focus={focus}
-      {...rest}
-      onFocus={event => {
-        setFocus(true);
-        if (onFocus) onFocus(event);
-      }}
-      onBlur={event => {
-        setFocus(false);
-        if (onBlur) onBlur(event);
-      }}
-      type="range"
-    />
-  );
-});
+const RangeInput = forwardRef(
+  ({ name, onChange, onFocus, onBlur, value: valueProp, ...rest }, ref) => {
+    const formContext = useContext(FormContext);
```

Comment:
Not sure it makes sense to me that RangeInput is using FormContext.

Reply:
RangeInput is just a `<input type="range" />`, which provides a numeric input control. So, it sets a numeric property value.